src/Vatras/Translation/Lang/2CC-to-ADT.agda

```diff
@@ -24,8 +24,10 @@ open import Relation.Binary.PropositionalEquality as Eq using (_≡_; refl; _≗
 open Eq.≡-Reasoning using (step-≡-⟨; step-≡-⟩; step-≡-∣; _∎)
 open IndexedSet using (_≅[_][_]_; ≅[]-sym; ≗→≅[])
 
-open import Vatras.Lang.2CC as 2CC using (2CC; 2CCL)
-open import Vatras.Lang.ADT as ADT using (ADT; ADTL; leaf; _⟨_,_⟩)
+open import Vatras.Lang.All
+open 2CC using () renaming (2CC to 2CCSyntax) -- Necessary for disambiguation
+open 2CC using (2CC; 2CCL)
```

Comment:
Why is `2CC` renamed to `2CCSyntax` but then imported again as `2CC`? Is this a bug?

Reply:
This import still does not look very intuitive. :thinking: